src/main/java/com/mycompany/app/App.java

```diff
@@ -9,5 +9,6 @@
     public static void main( String[] args )
     {
         System.out.println( "Hello World!" );
+        // No bugs found, but I'm going to push and open a pull request.
```

Comment:
Almost forgot to suggest changes...

Reply:
Fixed. If it is good now, please approve my pul request.